test/ex_sdp/attribute/ssrc_group_test.exs

```diff
@@ -0,0 +1,22 @@
+defmodule ExSDP.Attribute.SSRCGroupTest do
+  use ExUnit.Case, async: true
+
+  alias ExSDP.Attribute.SSRCGroup
+
+  describe "Flow identification - RTX SSRC" do
+    setup do
+      %{
+        attr_string: "FID 2231627014 632943048",
+        struct: %SSRCGroup{semantics: "FID", ssrcs: [2_231_627_014, 632_943_048]}
+      }
+    end
+
+    test "parsing", ctx do
+      assert SSRCGroup.parse(ctx.attr_string) == {:ok, ctx.struct}
+    end
+
+    test "serialization", ctx do
+      assert to_string(ctx.struct) == "ssrc-group:" <> ctx.attr_string
+    end
+  end
```

Comment:
Don't we need to test with invalid data, to check if ExSDP doesn't crash with invalid data?

Reply:
On one hand - we should, on the other, the errors are so crappy (https://github.com/membraneframework/ex_sdp/issues/17) it barely makes sense to use non-! variant. But I guess for completeness and to remember to change them later I should add them